domain/src/main/java/tachiyomi/domain/manga/interactor/GetUpcomingManga.kt

```diff
@@ -0,0 +1,19 @@
+package tachiyomi.domain.manga.interactor
+
+import eu.kanade.tachiyomi.source.model.SManga
+import tachiyomi.domain.manga.model.Manga
+import tachiyomi.domain.manga.repository.MangaRepository
+
+class GetUpcomingManga(
+    private val mangaRepository: MangaRepository,
+) {
+
+    private val includedStatuses = setOf(
+        SManga.ONGOING.toLong(),
+        SManga.PUBLISHING_FINISHED.toLong(),
+    )
+
+    suspend fun await(): List<Manga> {
```

Comment:
Make it a flow and subscribe to it
```suggestion
    suspend fun await(): List<Manga> {
```